app/routes/api/v1/warehouse/orders/by-id/index.ts

```diff
@@ -0,0 +1,92 @@
+import { OrderStatus } from "@prisma/client";
+import { ActionFunction, LoaderFunction } from "@remix-run/node";
+import { z } from "zod";
+import { badRequest, methodNotAllowed } from "~/helpers/app-helpers.server";
+import { parseBody } from "~/lib/parse-body.server";
+import {
+  createOrder,
+  deleteOrder,
+  updateOrder,
+} from "~/models/order/order.server";
+
+export const loader: LoaderFunction = async () => {
+  throw badRequest();
+};
+
+export const action: ActionFunction = async ({ request }) => {
+  switch (request.method.toLowerCase()) {
+    case "post": {
+      await handlePOSTRequest(request);
+    }
+
+    case "patch": {
+      await handlePATCHRequest(request);
+    }
+
+    case "delete": {
+      await handleDELETERequest(request);
+    }
```

Comment:
⚠️ 

The `case`s should return or throw the response, replace the `await` keyword